administration/src/cards/__tests__/CsvFactory.test.ts

```diff
@@ -0,0 +1,66 @@
+import { DynamicActivationCode } from '../../generated/card_pb'
+import bayernConfig from '../../project-configs/bayern/config'
+import nuernbergConfig from '../../project-configs/nuernberg/config'
+import CardBlueprint from '../CardBlueprint'
+import { CsvError, generateCsv, getCSVFilename } from '../CsvFactory'
+import { CreateCardsResult } from '../createCards'
+import NuernbergPassIdExtension from '../extensions/NuernbergPassIdExtension'
+import { findExtension } from '../extensions/extensions'
+
+jest.mock('csv-stringify/browser/esm/sync', () => ({
+  stringify: (input: any[]) => input[0].join(','),
+}))
+
+jest.mock('../../project-configs/showcase/config')
```

Comment:
why the showcase config is needed here?

Reply:
It is loaded in the getProjectConfig.ts and i did not want to wrap the components with the ProjectConfigProvider, as this will fallback to showcase in tests and I needed to specify them.